### PR TITLE
Index design

### PR DESCRIPTION
**Pull request by mihais98**

### CODE REVIEW

File: BusinessTripApplication/Web.config [hunk 1]
```diff
@@ -82,7 +82,7 @@
     </compilers>
   </system.codedom>
   <connectionStrings>
-    <add name="UserContext" connectionString="Data Source=(LocalDB)\MSSQLLocalDB;Integrated Security=True" providerName="System.Data.SqlClient" />
+    <add name="BusinessContext" connectionString="Data Source=(LocalDB)\MSSQLLocalDB;Integrated Security=True" providerName="System.Data.SqlClient" />
```
Comment (AlexCernov) [inline BusinessTripApplication/Web.config]: You should resolve the conflicts regarding the name of the db: please rename BussinesContext everywhere in your project ( it should be DatabaseContext)

File: BusinessTripApplication/Content/bootstrap.css [hunk 2]
```diff
@@ -1,81 +1,154 @@
 ﻿/*!
- * Bootstrap v3.3.7 (http://getbootstrap.com)
- * Copyright 2011-2016 Twitter, Inc.
+ * Bootswatch v4.1.2
```
Comment (AlexCernov) [inline BusinessTripApplication/Content/bootstrap.css]: Bootstrap files should not be modified. If you want to make changes to bootstrap classes you should override them in separate files.

**Comment by AlexCernov** [comment]
Please solve conflicts

File: BusinessTripApplication/Migrations/201807231229210_NewDbTables.resx [hunk 3]
```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (dragoscojanu97) [inline BusinessTripApplication/Migrations/201807231229210_NewDbTables.resx]: Nu trebuiesc adaugate .resx

**Pull request closed without merging**